lib/kernel/doc/guides/socket_usage.md

```diff
@@ -524,3 +524,69 @@ Options for level `sctp`:
 | rtoinfo           | sctp_rtoinfo()         | yes | yes | none                            |
 
 _Table: sctp options_
+
+
+[](){: #socket_configure_flags }
+
+## Socket Configure Flags
+
+There are a couple of configure flags, that can be used when (configure and)
+building Erlang/OTP, which effect the functionality of the 'socket' nif.
+
+### Builtin Socket Support
+
+Support for the builtin 'socket' (as a nif) can be explicitly enabled and
+disabled:
+
+```text
+--enable-esock (default) | --disable-esock
+```
+
+### RCVTIMEO/SNDTIMEO socket options
+
+Support for these (socket) options has to be explicitly enabled (see why above):
+
```

Comment:
I do not understand what part of above motivates this!?

Reply:
The explanation is in the options table in the 'Socket Options' chapter (but I do not know
of a way to link *into* a table. But a better "reference" is clearly needed.

Reply:
Suggestion:
"Support for these (socket) options has to be explicitly enabled.
 For details see the specific option descriptions
in the [socket option](socket_usage.md#socket_options_socket) "